modules/hal_nordic/nrfx/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+zephyr_library()
+zephyr_library_property(ALLOW_EMPTY TRUE)
+
+# Inherit key properties from the upstream nrfx library.
+zephyr_library_get_current_dir_lib_name(${ZEPHYR_NRF_MODULE_DIR} zephyr_lib_nrfx)
+zephyr_library_include_directories($<TARGET_PROPERTY:${zephyr_lib_nrfx},INCLUDE_DIRECTORIES>)
+zephyr_library_compile_definitions($<TARGET_PROPERTY:${zephyr_lib_nrfx},COMPILE_DEFINITIONS>)
+
+# Definitions copied from Zephyr.
+set(INC_DIR ${NRFX_DIR}/drivers/include)
+set(SRC_DIR ${NRFX_DIR}/drivers/src)
+set(MDK_DIR ${NRFX_DIR}/mdk)
+set(HELPERS_DIR ${NRFX_DIR}/helpers)
```

Comment:
but why are you copying those if you don't use them (except for the `MDK_DIR`) ?

And for the `MDK_DIR`, why is that in upper case ?